cpp/src/arrow/ipc/reader.cc

```diff
@@ -61,7 +61,7 @@ namespace {
 
 Status InvalidMessageType(Message::Type expected, Message::Type actual) {
   return Status::IOError("Expected IPC message of type ", FormatMessageType(expected),
-                         " got ", FormatMessageType(actual));
+                         " but got ", FormatMessageType(actual));
```

Comment:
FTR, I thought "expected XXX, got YYY" was quite common for error messages? Is it incorrect?